src/gsy_e/models/market/future.py

```diff
@@ -71,13 +142,6 @@ def market_time_slots(self) -> List[DateTime]:
         """Return list of all time slots of future markets."""
         return list(self.slot_bid_mapping.keys())
 
-    def open_bids_and_offers(self, **kwargs) -> Tuple[List, List]:
```

Comment:
This method seems to be outdated, there are no usages for it

Reply:
Wow, then the only usage if `self. slot_bid_mapping` and `self. slot_offer_mapping` is gone AFAIS. I wonder why we added them in the first place.
@spyrostz I see that we only export the trades (or statistics of them) for the future markets. I remember that we decided to postpone the export of the bids and offers per future market slot until somebody asks for it right?

If all the above is true, I wonder why the current change is needed. I was under the impression that we needed this to fix a bug because of not synced buffers. But as the order mappings are not used somewhere else, why is this needed @Hassan754 ?

Reply:
@hannesdiedrich we're using these mappings in `orders_per_slot`, `get_offers_per_slot` and `get_bids_per_slot` .

I decided to introduce this feature when debugging a future market issue, in the end it turned out that this sync isn't the problem, but it's a nice and secure to add improvement to our codebase, and as you can see this saved us lots of overriding methods just for the sake of also adding the order to the slot_offer_mapping and slot_bid_mapping

Reply:
Ah I see now, `orders_per_slot` is really the only place where we need them and it is important for the matching algorithm, right? Then ignore me.
BTW; `get_bids_per_slot ` and `get_offers_per_slot` do not seem to be used anywhere.

Reply:
Interesting! 2 more methods to be removed !